tests/Integration/ValidationTest.php

```diff
@@ -522,6 +522,27 @@ public function testCombinesArgumentValidationByPausingAndResuming(): void
         );
     }
 
+    public function testCombinesArgumentValidationWhenGrouped(): void
+    {
+        $this->schema .= '
```

Comment:
```suggestion
        $this->schema .= /* @lang GraphQL */ '
```

Reply:
This isn't used anywhere. I'm simply copying codestyle from the code above.

Reply:
We are working towards including those annotations. Please add them in parts of code you touched.